### PR TITLE
☎️  Customer Support | add accessibility/ADA checkbox

### PR DESCRIPTION
**Pull request by i0sea**
#### Summary of changes

**Asana Ticket:** [☎️  Customer Support | add accessibility/ADA checkbox](https://app.asana.com/0/385363666817452/1204160843045634/f)

---
Added checkbox to customer feedback form to allow customers to label feedback as an accessibility or ADA concern. If labeled as such can be filtered to correct channels.

Note: just the commits with ☎️ are related to this work, commit history got a little messy

#### General checks
* [ ] **Are the changes organized into self-contained commits with descriptive and well-formatted commit messages?** This is a good practice that can facilitate easier reviews.
* [x] **Testing.** Do the changes include relevant passing updates to tests? This includes updating screenshots. Preferably tests are run locally to verify that there are no test failures created by these changes, before opening a PR.
* [ ] **Tech debt.** Have you checked for tech debt you can address in the area you're working in? This can be a good time to address small issues, or create Asana tickets for larger issues.

### CODE REVIEW

**Comment by arkadyan** [comment]
Is this a duplicate of https://github.com/mbta/dotcom/pull/1553/files?

**Comment by thecristen** [comment]
@arkadyan Yeah, this is the feature branch containing #1553 + #1545 + #1540, which have already been reviewed.

**Comment by thecristen** [comment]
Sorry for the confusion - @KaylaBrady feel free to look at the linked PRs in my prior comment. This PR is here for merging the master branch into main. We're waiting on some stakeholder approval to merge, but feel free to give this a final sanity check!

File: apps/site/assets/css/_customer-support.scss
Comment (KaylaBrady) [inline apps/site/assets/css/_customer-support.scss:265]: question (non-blocking): What does the `isa-` mean here? Is it needed?

File: apps/site/assets/css/_customer-support.scss
Comment (KaylaBrady) [inline apps/site/assets/css/_customer-support.scss:261]: question: I'm not sure if there is a design file I should look at - is this the expected layout with the icon on small screens?
image

**Review by KaylaBrady — COMMENTED** [review]
Looks good! Focused my review on the last commit which was not a part of previously reviewed PRs. Just one minor layout question!

File: apps/site/assets/css/_customer-support.scss
Comment (i0sea) [reply apps/site/assets/css/_customer-support.scss:265]: ISA is the technical name of this icon ♿  Can change to ada to remain consistent with naming throughout pr

Comment (KaylaBrady) [reply apps/site/assets/css/_customer-support.scss:265]: TIL, thank you! I have no strong preference for changing it then if it is already a conventional name.

File: apps/site/assets/css/_customer-support.scss
Comment (i0sea) [reply apps/site/assets/css/_customer-support.scss:261]: @KaylaBrady we'll create a separate pr for this particular change

**Review by KaylaBrady — APPROVED** [review]

**Pull request merged**
